### PR TITLE
update of verification for HMI result code VEHICLE_DATA_NOT_AVAILABLE

### PR DESCRIPTION
**Pull request by istoimenova**

### CODE REVIEW

**Comment by istoimenova** [comment]
@OVikhrov, @dboltovskyi, please check update based on VEHICLE_DATA_NOT_AVAILABLE.

File: test_scripts/API/IsReady/UI_IsReady/SetAudioStreamingIndicator/002_ATF_N_TC_SetAudioStreamingIndicator_success_false_IsReady_HMI_doesnot_reply.lua [hunk 1]
```diff
@@ -109,8 +109,12 @@ for i = 1, #hmi_result_code do
 
 	  EXPECT_HMICALL("UI.SetAudioStreamingIndicator", { audioStreamingIndicator = "PAUSE" })
 	  :Do(function(_,data)
-			-- ATF issue: "Genivi: SDL doesn't accept some error_codes from HMI when they are sending with type of protocol_message "error""
-			self.hmiConnection:SendError(data.id, data.method, hmi_result_code[i].result_code, "error message")
+			--TODO (istoimenova): If should be removed when "[ATF] ATF doesn't process code of HMI response VEHICLE_DATA_NOT_AVAILABLE in error message." is fixed.
+	  	if(hmi_result_code[i].result_code == "VEHICLE_DATA_NOT_AVAILABLE") then 
+	  		self.hmiConnection:Send('{"error":{"data":{"method":"UI.SetAudioStreamingIndicator"},"message":"error message","code":9},"jsonrpc":"2.0","id":'..tostring(tostring(data.id))..'}')
```
Comment (dboltovskyi) [inline test_scripts/API/IsReady/UI_IsReady/SetAudioStreamingIndicator/002_ATF_N_TC_SetAudioStreamingIndicator_success_false_IsReady_HMI_doesnot_reply.lua]: @istoimenova, Please remove 2nd usage of 'tostring()' function

File: test_scripts/API/SetAudioStreamingIndicator/033_ATF_N_TC_SetAudioStreamingIndicator_media_app_HMI_replies_resultcode_success_false.lua [hunk 2]
```diff
@@ -87,8 +87,12 @@ for i = 1, #hmi_result_code do
 
 	  EXPECT_HMICALL("UI.SetAudioStreamingIndicator", { audioStreamingIndicator = "PAUSE" })
 	  :Do(function(_,data) 
-	  	-- ATF issue: "Genivi: SDL doesn't accept some error_codes from HMI when they are sending with type of protocol_message "error""
-	  	self.hmiConnection:SendError(data.id, data.method, hmi_result_code[i].result_code, "error message") 
+	  	--TODO (istoimenova): If should be removed when "[ATF] ATF doesn't process code of HMI response VEHICLE_DATA_NOT_AVAILABLE in error message." is fixed.
+	  	if(hmi_result_code[i].result_code == "VEHICLE_DATA_NOT_AVAILABLE") then 
+	  		self.hmiConnection:Send('{"error":{"data":{"method":"UI.SetAudioStreamingIndicator"},"message":"error message","code":9},"jsonrpc":"2.0","id":'..tostring(tostring(data.id))..'}')
```
Comment (dboltovskyi) [inline test_scripts/API/SetAudioStreamingIndicator/033_ATF_N_TC_SetAudioStreamingIndicator_media_app_HMI_replies_resultcode_success_false.lua]: @istoimenova, Please remove 2nd usage of 'tostring()' function

File: test_scripts/API/SetAudioStreamingIndicator/067_ATF_N_TC_SetAudioStreamingIndicator_media_app_sends_2_requests_HMI_replies_resultcode_success_false.lua [hunk 3]
```diff
@@ -93,7 +93,12 @@ for i = 1, #hmi_result_code do
 	  	corr_id_second_req = self.mobileSession:SendRPC("SetAudioStreamingIndicator", { audioStreamingIndicator = "PAUSE" })
 	  	
 	  	local function send_HMI_response()
-	  		self.hmiConnection:SendError(data.id, data.method, hmi_result_code[i].result_code, "error message") 
+		  	--TODO (istoimenova): If should be removed when "[ATF] ATF doesn't process code of HMI response VEHICLE_DATA_NOT_AVAILABLE in error message." is fixed.
+		  	if(hmi_result_code[i].result_code == "VEHICLE_DATA_NOT_AVAILABLE") then 
+		  		self.hmiConnection:Send('{"error":{"data":{"method":"UI.SetAudioStreamingIndicator"},"message":"error message","code":9},"jsonrpc":"2.0","id":'..tostring(tostring(data.id))..'}')
```
Comment (dboltovskyi) [inline test_scripts/API/SetAudioStreamingIndicator/067_ATF_N_TC_SetAudioStreamingIndicator_media_app_sends_2_requests_HMI_replies_resultcode_success_false.lua]: @istoimenova, Please remove 2nd usage of 'tostring()' function

File: test_scripts/API/IsReady/UI_IsReady/SetAudioStreamingIndicator/002_ATF_N_TC_SetAudioStreamingIndicator_success_false_IsReady_HMI_doesnot_reply.lua (hunk 1 above)
Comment (istoimenova) [inline test_scripts/API/IsReady/UI_IsReady/SetAudioStreamingIndicator/002_ATF_N_TC_SetAudioStreamingIndicator_success_false_IsReady_HMI_doesnot_reply.lua]: @dboltovskyi, done in fd8ba9fed9bd1025bd46c709b948f3a4622fce8f

File: test_scripts/API/SetAudioStreamingIndicator/033_ATF_N_TC_SetAudioStreamingIndicator_media_app_HMI_replies_resultcode_success_false.lua (hunk 2 above)
Comment (istoimenova) [inline test_scripts/API/SetAudioStreamingIndicator/033_ATF_N_TC_SetAudioStreamingIndicator_media_app_HMI_replies_resultcode_success_false.lua]: @dboltovskyi, done in fd8ba9fed9bd1025bd46c709b948f3a4622fce8f

File: test_scripts/API/SetAudioStreamingIndicator/067_ATF_N_TC_SetAudioStreamingIndicator_media_app_sends_2_requests_HMI_replies_resultcode_success_false.lua (hunk 3 above)
Comment (istoimenova) [inline test_scripts/API/SetAudioStreamingIndicator/067_ATF_N_TC_SetAudioStreamingIndicator_media_app_sends_2_requests_HMI_replies_resultcode_success_false.lua]: @dboltovskyi, done in fd8ba9fed9bd1025bd46c709b948f3a4622fce8f

**Pull request merged**